modules/apps/foundation/frontend-theme/space-theme/src/templates/navigation.ftl

```diff
@@ -0,0 +1,61 @@
+
+<nav class="${nav_css_class}" id="navigation" role="navigation"  class="navbar navbar-default navbar-fixed-top">
+	<div class="container-fluid">
+			<h1 class="hide-accessible"><@liferay.language key="navigation" /></h1>
+			<h1 class="site-title">
+					<a class="${logo_css_class}" href="${site_default_url}" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+						<img alt="${logo_description}" height="${site_logo_height}" src="${site_logo}" width="${site_logo_width}" />
+					</a>
+		
+					<#if show_site_name>
+						<span class="site-name" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+							${site_name} 
+						</span>
+					</#if> 
+			</h1>
```

Comment:
Add missing line break.  Line break required between block level elements in HTML.